nucleus/dataset_item.py

```diff
@@ -122,24 +122,16 @@ def __post_init__(self):
         )
 
     @classmethod
-    def from_json(cls, payload: dict, is_scene=False):
+    def from_json(cls, payload: dict):
```

Comment:
Is removing this an argument an API breaking change?

Reply:
This is almost entirely used internally so I don't think so